pkg/plugins/common/kustomize/v1/init.go

```diff
@@ -94,6 +107,50 @@ func (p *initSubcommand) InjectConfig(c config.Config) error {
 	return nil
 }
 
+func (p *initSubcommand) PreScaffold(machinery.Filesystem) error {
+
+	// Check what is the SO
+	cmd := exec.Command("uname", "-s")
+	os, err := cmd.CombinedOutput()
+	osFound := strings.ToLower(string(os))
+	if err != nil {
+		log.Warnf("unable to check the Operational System to verify if kustomize/v1 can be supported "+
+			"on this environment: %s", err)
+		return nil
+	}
+	// Check what is the ARCH
+	cmd = exec.Command("uname", "-m")
+	arch, err := cmd.CombinedOutput()
+	archFound := strings.ToLower(string(arch))
+	archFound = strings.Replace(archFound, "x86_64", "amd64", -1)
+	if err != nil {
+		log.Warnf("unable to check the architecture to verify if kustomize/v1 can be supported on "+
+			"this environment: %s", err)
+		return nil
+	}
+	localPlatform := fmt.Sprintf("%s/%s", strings.TrimSpace(osFound),
+		strings.TrimSpace(archFound))
+
```

Comment:
It might be easiest to do something like:
```go
localPlatform := fmt.Sprintf("%s/%s", strings.TrimSpace(runtime.GOOS), strings.TrimSpace(runtime.GOARCH))
```
This way you can get rid of lines 112 - 130.

WDYT?

Reply:
The replacement for the goarch is still required but yep. We follow up on your suggestion here as well.
Great !